FXyz-Importers/src/main/java/org/fxyz3d/importers/obj/ObjImporter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2010, 2014, Oracle and/or its affiliates.
```

Comment:
This is a big refactoring. It looks fine, but have you tried loading different samples (obj/polyObj)?

Reply:
I tried loading my own obj files. They load fine.